chapters/derivationofstream.tex

```diff
@@ -116,22 +108,17 @@ \section{Rationale for inStream}\label{rationale-for-the-formulation-of-the-inst
 h_{\mathrm{mix}}=\frac{\operatorname{max}(-\tilde{m}_1,0)h_{\mathrm{outflow},1}+\operatorname{max}(-\tilde{m}_2,0)h_{\mathrm{outflow},2}+\operatorname{max}(-\tilde{m}_3,0)h_{\mathrm{outflow},3}}
 {\operatorname{max}(-\tilde{m}_1,0)+\operatorname{max}(-\tilde{m}_2,0)+\operatorname{max}(-\tilde{m}_3,0)}
 \end{equation*}
-Above it was shown that an equation of this type does not yield properly
-formulated model equations. In the streams concept we therefore decide
-to split the energy balance, which consists of different branches
-depending on the mass flow direction. Consequently, separate energy
-balances are the result; each valid for specific flow directions.
+Above it was shown that an equation of this type does not yield properly formulated model equations.
+In the streams concept we therefore decide to split the energy balance, which consists of different branches depending on the mass flow direction.
+Consequently, separate energy balances are the result; each valid for specific flow directions.
 
-In a model governing equations have to establish the specific enthalpy
-of fluid leaving the model based on the specific enthalpy of fluid
-flowing into it. Whenever the mixing enthalpy is \emph{used} in a model
-it is therefore the mixing enthalpy under the assumption of fluid
-flowing into said model.
+In a model governing equations have to establish the specific enthalpy of fluid leaving the model based on the specific enthalpy of fluid flowing into it.
+Whenever the mixing enthalpy is \emph{used} in a model it is therefore the mixing enthalpy under the assumption of fluid flowing into said model.
 
-We establish this quantity using a dedicated operator $\text{\lstinline!inStream!}(h_{\mathrm{outflow},i})=h_{\mathrm{mix}}$ assuming that $\tilde{m}_{i} \geq 0$. This leads to
-three different incarnations of ($n$ in the general case). This is
-illustrated in the figure below. For the present example of three
-components in a connection set, this means the following.
+We establish this quantity using a dedicated operator $\text{\lstinline!inStream!}(h_{\mathrm{outflow},i})=h_{\mathrm{mix}}$ assuming that $\tilde{m}_{i} \geq 0$.
+This leads to three different incarnations of ($n$ in the general case).
```

Comment:
Restoring the missing math it would be:
```suggestion
This leads to three different incarnations of $h_{\mathrm{mix}}$ ($n$ in the general case).
```
However, 
* I think we should review the entire chapter separately.
* The text isn't easy to understand, I would propose something like the text below.

```suggestion
This leads to different incarnations of $h_{\mathrm{mix}}$, three in this case and $n$ in the general case.
```